app/car.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+
+class Car:
+    cars = []
+
+    def __init__(
+            self,
+            brand: str,
+            fuel_consumption: float
+    ) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    @classmethod
+    def add_to_car(cls) -> None:
+        with open("app/config.json", "r") as cars_file:
+            customer_data = json.load(cars_file)
+            for customer in customer_data["customers"]:
+                for key, value in customer.items():
+                    if key == "car":
```

Comment:
you can access car as 
customer["car"] no need to iterate for key, value
also key and value are so bad names